specification/sources/chapters/instance.adoc

```diff
@@ -181,7 +181,15 @@ different results if a pname:layerName is available in one call but not in
 another.
 The extensions supported by a layer may also change between two calls, e.g.
 if the layer implementation is replaced by a different version between those
-calls.
+calls, or if an `XrInstance` is created or destroyed between those calls.
+
+This function may: be called before an instance has been created;
+implementations must: not assume an instance exists.
+
+When applications invoke this function, the OpenXR loader's implementation
+is used, which enumerates the registered runtime and API layer manifests;
+this function must: still be implemented by API layers and runtimes, as it may
```

Comment:
This is a new unenforceable `must:` - it was already required that runtimes implemented this function, but it wasn't required for API layers. Is there some path to make this enforceable?

Reply:
I don't think this is new for API layers - it's already a `must:` at https://github.com/KhronosGroup/OpenXR-SDK-Source/blob/main/specification/loader/api_layer.adoc#api-layer-conventions-and-rules

Reply:
As for making it enforceable:
- we don't currently have any CTS that's really usable for API layers. Perhaps this should change?
- we *could* make the loader refuse to load extensions that do not implement the functions and where the detail doesn't matter. This would have a minor performance cost on startup.

Reply:
It's not new via the loader spec; but the loader spec is full of things like this. What's being proposed here is a change to the OpenXR spec where we are much more careful. You can see the difference in style / enforcement in the MRs / PRs where I moved the negotiate functions from the loader spec to the OpenXR spec.

Reply:
> I don't think this is new for API layers - it's already a must: 

My assumption here is that the 'must:' for API layers in the loader specification are actually requirements for API layers; is this the actually the case, or is this just describing implementations that Khronos's loader implementation is allowed to make?

If the latter, that implies there's potentially such a thing as a conformant API layer that the Khronos loader is unable to load - that seems undesirable.